.github/workflows/maintenance.yml

```diff
@@ -0,0 +1,139 @@
+        runs-on: ubuntu-latest
+        env:
+            GPG_SIGNING_KEY: ${{ secrets.GPG_SIGNING_KEY }}
+            GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        strategy:
+            fail-fast: false
+            matrix:
+                test: ${{ fromJson(needs.prepare-maintenance.outputs.test-infos) }}
+        steps:
+            - uses: actions/checkout@v4
+              with:
+                  ref: ${{ needs.create-branch-prefix.outputs.base-branch }}
+                  token: ${{ secrets.LASTMJS_GITHUB_TOKEN }}
+
+            - uses: ./.github/actions/setup_node
+
+            # Update package-lock.json in test directory
+            - name: Update package-lock.json
+              working-directory: ${{ matrix.test.path }}
+              run: |
+                  rm -f package-lock.json
+                  npm install
+
+            - name: Create branch name
+              id: create-branch-name
+              uses: ./.github/actions/create_branch_name
+              with:
+                  prefix: ${{ needs.create-branch-prefix.outputs.branch-prefix }}
+                  path: ${{ matrix.test.displayPath }}
+
+            - uses: ./.github/actions/commit_and_push
+              with:
+                  branch-name: ${{ steps.create-branch-name.outputs.branch-name }}
+                  commit-message: 'Update dependencies for ${{ matrix.test.displayPath }}'
+                  gpg-signing-key: ${{ secrets.GPG_SIGNING_KEY }}
+                  create-branch: 'true'
+                  signing-key-id: ${{ inputs.signing-key-id }}
+
+    squash-branches:
+        needs:
+            - prepare-maintenance
+            - update-test-dependencies
+            - create-branch-prefix
+        uses: ./.github/workflows/squash_branches.yml
+        with:
+            base-branch: ${{ needs.create-branch-prefix.outputs.base-branch }}
+            branch-prefix: ${{ needs.create-branch-prefix.outputs.branch-prefix }}
+            commit-message: 'Maintenance: Update dependencies and fix formatting'
+            signing-key-id: ${{ inputs.signing-key-id }}
+        secrets:
+            GPG_SIGNING_KEY: ${{ secrets.GPG_SIGNING_KEY }}
+            GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+            LASTMJS_GITHUB_TOKEN: ${{ secrets.LASTMJS_GITHUB_TOKEN }}
+
+    create-pr:
+        needs:
+            - prepare-maintenance
+            - squash-branches
+            - create-branch-prefix
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v4
+              with:
+                  ref: ${{ github.ref }}
+
+            - name: Create Pull Request
+              env:
+                  GH_TOKEN: ${{ secrets.LASTMJS_GITHUB_TOKEN }}
+              run: |
+                  gh pr create \
+                    --base main \
+                    --head ${{ needs.create-branch-prefix.outputs.base-branch }} \
+                    --title "Maintenance: Update dependencies and fix formatting" \
```

Comment:
Change all title, names, commit messages to the same thingish

Reply:
Done